polaris-react/src/components/LegacyFilters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -4,7 +4,7 @@
 // stylelint-disable selector-max-specificity -- generated by polaris-migrator DO NOT COPY
 // stylelint-disable selector-max-compound-selectors -- generated by polaris-migrator DO NOT COPY
 // stylelint-disable selector-max-type -- generated by polaris-migrator DO NOT COPY
-.ConnectedFilterControl {
```

Comment:
This intentional?

Reply:
no it got deleted somehow and I couldn't figure out why the build was failing 😅 thanks for catching